docker/scripts/deprovision.sh

```diff
@@ -64,7 +64,7 @@ function autofail() {
 trap autofail EXIT
 
 # Check BIOS config and update if drift is detected
-if [[ $arch == x86_64 ]] && [[ $reserved != "true" ]]; then
+if [[ $arch == "x86_64" ]] && [[ $reserved != "true" ]]; then
```

Comment:
Quote your variables here too to avoid expansion hell.

Reply:
Done (also fixed throughout the file for consistency). For whatever reason, shellcheck isn't complaining about this, so it's been missed.